packages/common/src/types/warehouse.ts

```diff
@@ -24,14 +29,27 @@ export interface WarehouseClient {
         }[],
     ) => Promise<WarehouseCatalog>;
 
+    streamQuery(
+        query: string,
+        streamCallback: (data: WarehouseResults) => void,
```

Comment:
just curious. do we know roughly how big are the chunks per warehouse ? is it random ? is it a fixed number ? are they consistent per warehouse ?

Reply:
If I'm not mistaken, this is configurable in the warehouse. Example: https://docs.databricks.com/en/structured-streaming/batch-size.html